rdf/rdf12/rdf-semantics/manifest.ttl

```diff
@@ -102,7 +102,7 @@ trs:annotated-asserted a mf:PositiveEntailmentTest;
   test:approval test:NotClassified .
 
 trs:annotation a mf:PositiveEntailmentTest;
-  rdfs:comment "Annotation are about the reifying triple.";
+  rdfs:comment "Annotations are about the reifying triple.  This is about shorthand expansion, and is not really a semantics test.";
```

Comment:
I think this (and similar) is in error. `the reifying triple` is the triple with `rdfs:reifies` predicate. `the reified triple` is the one about which an annotation is written.
```suggestion
  rdfs:comment "Annotations are about the reified triple.  This is about shorthand expansion, and is not really a semantics test.";
```